erpnext/payroll/doctype/additional_salary/additional_salary.json

```diff
@@ -7,25 +7,30 @@
  "editable_grid": 1,
  "engine": "InnoDB",
  "field_order": [
+  "employee_details_section",
   "naming_series",
   "employee",
   "employee_name",
-  "salary_component",
-  "type",
-  "amount",
-  "ref_doctype",
-  "ref_docname",
-  "amended_from",
   "column_break_5",
   "company",
   "department",
```

Comment:
`department` field should be below `employee_name` since it is fetched from the `employee` field.

Reply:
yeah, that was in my mind but. There will be three fields in a single column having lots of (employee_name, department, and employee), which will leave only one field for second column. causing lots of white space in the second column. like three fields in the first column and only single fields in the second column, but anyway **they are in the same section** (**EmPloyee details**).